---
title: Falha no teste de desempenho do Cloud Manager Assets
description: Saiba por que o teste de desempenho do Cloud Manager Assets está falhando.
solution: Experience Manager
product: Experience Manager
applies-to: Experience Manager
keywords: KCS, gerenciador de nuvem, desempenho de ativos, erro, Experience Manager
resolution: Resolution
internal-notes: https://jira.corp.adobe.com/browse/CQDOC-21090
bug: false
article-created-by: Tanay Sharma .
article-created-date: 12/5/2023 3:46:03 PM
article-published-by: Tanay Sharma .
article-published-date: 12/5/2023 4:22:43 PM
version-number: 2
article-number: KA-23312
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=ed605461-8593-ee11-be37-6045bd006b25
exl-id: 546d28f5-8d3a-45fa-bfef-82a31b68e85e
source-git-commit: f0598a17997f23ee5ce25d90c6488c22ca4838a1
workflow-type: tm+mt
source-wordcount: '280'
ht-degree: 1%

---

# Falha no teste de desempenho do Cloud Manager Assets


O teste de desempenho do Assets está falhando com um erro desconhecido. Saiba por que isso está acontecendo.

## Descrição {#description}


### Ambiente

AEM Managed Services

### Problema/Sintomas 

O teste de desempenho do Assets está falhando com um erro desconhecido.

<b>Falha da etapa</b>: assetsTest

Nenhum registro direto do AssetsTest da etapa que falhou está disponível, conforme indicado pela mensagem de erro:

`{"type":"http://ns.adobe.com/adobecloud/missing-step-logs","status":404,"title":"Step logs not found","errors":[ "Logs not found for programId=809, executionId=3473820, phaseId=6253498, stepId=11687695"] } `

No entanto, pipelineExecutionFailureReasons indica um `ASSETS_TESTS_AUTHOR_CONNECTION_ERROR` com um código de erro `002_010_006` e a mensagem:

`Unable to connect to the author instance. `

Detalhes internos sugerem um problema de resposta HTTP com uma mensagem indicando *Não encontrado*, o que normalmente implicaria que o endpoint da instância do autor não poderia ser acessado durante a execução do teste.

O problema principal parece ser um erro de conexão com a instância do autor durante a etapa assetsTest, que impediu a conclusão com êxito do teste.

No entanto, o usuário do Cloud Manager existe e o upload manual de ativos no autor funciona conforme esperado.

O teste de desempenho de ativos está funcionando por 30 minutos e, de repente, falha de maneira repetida, enquanto o teste deve parar após 30 minutos.

Consulte [AEM Assets](https://experienceleague.adobe.com/docs/experience-manager-cloud-manager/content/using/code-quality-testing.html#aem-assets)na Documentação do Cloud Manager para AEM 6.x.


## Resolução {#resolution}


Verificando os logs internos da função do Azure, descobriu-se que a solicitação tinha o resultado 404 Não encontrado. A GET para o URL do *mensagem* O campo retornou 404. Veja abaixo.

A falha nesse caso foi causada pelo fato de que, seguindo a recomendação do Adobe, o querybuilder foi bloqueado no nível do dispatcher.
No entanto, o Cloud Manager precisa que o querybuilder tenha permissão no Dispatcher para o usuário do Cloud Manager, pois, durante o teste do ativo, ele recupera o tempo de processamento.




```
{
 "x-ms-workflow-run-id": " ",
 "x-ms-workflow-operation-name": "1500_measureAssetsProcessingTime",
 "message": "Req: GET https://author-URL.adobecqms.net/bin/querybuilder.json?p.hitwriter=selective&p.limit=1000&p.properties=jcr%3Acontent%2Fjcr%3AlastModified+jcr%3Acreated&path=%2Fcontent%2Fdam%2Fcloudmanager&property=jcr%3Acontent%2Fdam%3AassetState&property.depth=1&property.value=processed&type=dam%3AAsset",
 "x-ms-client-tracking-id": "trackingID",
 "trackingContext": {
 "executionId": executionID,
 "tenantId": tenantID,
 "imsOrgId": "IMS_ORG",
 "programId": programID,
 "pipelineId": pipelineID
 }
}
```
